hack/toolchain.sh

```diff
@@ -15,14 +15,23 @@ tools() {
     go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.41.1
     go install github.com/google/ko@v0.11.2
     go install github.com/mikefarah/yq/v4@v4.16.1
-    go install github.com/mitchellh/golicense@v0.2.0
     go install github.com/norwoodj/helm-docs/cmd/helm-docs@v1.7.0
     go install github.com/onsi/ginkgo/ginkgo@v1.16.5
     go install sigs.k8s.io/controller-runtime/tools/setup-envtest@v0.0.0-20220113220429-45b13b951f77
     go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.7.0
     go install github.com/sigstore/cosign/cmd/cosign@v1.5.1
     go install github.com/gohugoio/hugo@v0.92.1+extended
 
+    # golicense no longer builds with go 1.18+, for now just install the last release binary
+    if [[ $(go env GOOS) == "darwin" ]]; then
```

Comment:
Can we wrap this with a helper like kubebuilder?

Reply:
Another option would be to keep an older version of go on the host and just use that for the install command.

Reply:
Just created a helper for now, I viewed this as an interim solution to allow us to move to Go 1.18 whenever we want to.  Hopefully the main repo will accept a PR or a fork will emerge that has this feature fixed.